cranelift/bitset/src/compound.rs

```diff
@@ -346,7 +346,7 @@ impl CompoundBitSet {
             .enumerate()
             .rev()
             .filter_map(|(word, sub)| {
-                let bit = sub.max()?;
+                let bit = sub.last()?;
```

Comment:
`ScalarBitset` now implements `Ord` (because pulley `Instr` derives `Ord`). So calls to `bitset.min()` will resolve to the `min` method from `Ord`, rather than the inherent method attached to `ScalarBitset`

Reply:
Can we avoid `Ord` for `ScalarBitSet`? And instead implement it for `RegSet`?

Reply:
We can remove the `#[derive(PartialOrd, Ord)]` on `Op`, since it's not used anywhere.